src/modules/App/mobile.scss

```diff
@@ -6,3 +6,14 @@
 .sb-show-main {
   padding: 0 !important;
 }
+
+.sb_mobile__panelwrap .sendbird-thread {
+  width: 100%;
+  height: 100%;
+  & .sendbird-thread-ui {
+    max-width: 100%;
+    & .sendbird-thread-ui__header {
+      width: 100%;
+    }
+  }
+}
```

Comment:
Shouldnt this better be:

in https://github.com/sendbird/sendbird-uikit-react/blob/feature/UIKIT-3476/Mobile-support-for-thread/src/modules/Thread/components/ThreadUI/index.scss#L3

```
  @include mobile() {
    width: 100%;
  }
```

Reply:
Ideally, we have to imagine they should work flawlessly even when used independently (ie.outside App(Mobile)).. Given a proper container